islandora.module

```diff
@@ -397,7 +397,14 @@ function islandora_menu() {
     'type' => MENU_NORMAL_ITEM,
     'access arguments' => array(ISLANDORA_MANAGE_DELETED_OBJECTS),
   );
-
+  $items['admin/reports/orphaned_objects/list'] = array(
+    'title' => 'Orphaned objects',
+    'description' => 'List of orphaned objects.',
```

Comment:
I would add the word Islandora here, since this goes into the admin reports menu

Reply:
Addressed with latest commit